### PR TITLE
new graphic readme

### PR DESCRIPTION
**Pull request by joel-hamill**
## Urgency
- [ ] Blocker 
- [x] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-docs#test-local) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, and 1.9).
- See the [contribution guidelines](https://github.com/dcos/dcos-docs#contributing).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ This page provides instructions on how to contribute to the DC/OS documentation.
 
 1. Search [JIRA](https://jira.mesosphere.com/) to review the currently open issues and make sure that no one is already working on your issue. If you find an open issue that is unassigned that you want to work, you can assign it to yourself! If you don’t see an issue related to yours, [create a new issue](https://jira.mesosphere.com/secure/CreateIssue!default.jspa), select documentation as the component, and assign it to yourself.
 
-  <img src="/images/2017-01-11_JIRA_screen_cap.png"/>
+  <img src="/images/2017-03-13_JIRA_screen_cap.png"/>
 
 1. If this is your first contribution [Fork](https://help.github.com/articles/fork-a-repo/) the [dcos-docs](https://github.com/dcos/dcos-docs) repo. (Once you’ve forked the repo, that fork stays associated with your GitHub account. If you try to fork it again GitHub will remind you that you already have a fork.)
```
Comment (ryadav88) [inline README.md]: the link in point 8 still points to https://dcosjira.atlassian.net/

File: README.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ This page provides instructions on how to contribute to the DC/OS documentation.
 
 1. Search [JIRA](https://jira.mesosphere.com/) to review the currently open issues and make sure that no one is already working on your issue. If you find an open issue that is unassigned that you want to work, you can assign it to yourself! If you don’t see an issue related to yours, [create a new issue](https://jira.mesosphere.com/secure/CreateIssue!default.jspa), select documentation as the component, and assign it to yourself.
 
-  <img src="/images/2017-01-11_JIRA_screen_cap.png"/>
+  <img src="/images/2017-03-13_JIRA_screen_cap.png"/>
```
Comment (ryadav88) [inline README.md]: image doesnt exist

File: README.md (hunk 1 above)
Comment (joel-hamill) [inline README.md]: fixed https://github.com/dcos/dcos-docs/pull/952/commits/5c33ffe6678f0a9af4bb4ed6533ee27ebf0e76d4

File: README.md (hunk 2 above)
Comment (joel-hamill) [inline README.md]: fixed https://github.com/dcos/dcos-docs/pull/952/commits/5c33ffe6678f0a9af4bb4ed6533ee27ebf0e76d4

**Pull request merged**